### DISCUSSION TITLE
Bulk order payments options

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#1104] [Ideas]
**Describe the bug**
In the course of time a busy restaurant will have many unpaid orders, meaning a customer has come he/she has placed an order, order has been placed to the kitchen, served and a bill has been produced, the customer has paid and left, the cashier takes cash and is left with the bill to clear later from the system, the cashier is busy does not clear the bills months later bills are still on hold. To clear the orders on hold would require bulk order payments options, where the cashier selects the unpaid orders for a particular waiter and makes payment.

**To Reproduce**
1. Go to 
![Add bulk payment Option](https://user-images.githubusercontent.com/13999977/194105718-ac089cc1-5513-43ef-9b84-30c52d4eddb5.png)

**Expected behavior**

![Add bulk payment Option - Copy](https://user-images.githubusercontent.com/13999977/194107626-a627d6a6-cb68-44c8-9337-e7fc1b5c1174.png)

**Screenshots**

**Environment:**
 - NexoPOS 4.x vesrion: [e.g. 4.8.5]

**Comment by Blair2004** [comment]
Hi, 
Thank you your suggestion is taken into account. However, a bulk modification is actually scheduled to be a premium module. This won't be merged to the core.
